src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/filtersAndSampling.tsx

```diff
@@ -154,45 +210,112 @@ class FiltersAndSampling extends AsyncView<Props, State> {
     );
   };
 
+  handleAddRule = <T extends keyof Pick<State, 'errorRules' | 'transactionRules'>>(
+    type: T
+  ) => () => {
+    if (type === 'errorRules') {
```

Comment:
Do you have an enum for this somewhere?

Reply:
No. I believe that enum here is not necessary